tests/test_pyglmnet.py

```diff
@@ -122,8 +122,10 @@ def test_group_lasso():
     beta = np.random.normal(0.0, 1.0, n_features)
     beta[groups == 2] = 0.
 
+    alpha = 1.0
+    reg_lambda = 0.2
     # create an instance of the GLM class
-    glm_group = GLM(distr='softplus', alpha=1.)
+    glm_group = GLM(distr='softplus', alpha=alpha, reg_lambda=reg_lambda, group=groups)
```

Comment:
can you make sure you conform to [pep8 conventions](https://www.python.org/dev/peps/pep-0008/)? You can add a plugin to your editor or install a command-line checker. Right now, the line exceeds 79 characters